Project.toml

```
@@ -47,7 +47,7 @@ CUDA = "4, 5"
 ChainRulesCore = "1.12"
 Compat = "4.10.0"
 Enzyme = "0.12, 0.13"
-Functors = "0.4"
+Functors = "0.4, 0.5"
```

Flux 0.15 should probably drop Functors 0.4, right? We want to document that `@functor` is no longer required.
```suggestion
Functors = "0.5"
```